irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -34,18 +31,22 @@ namespace iroha {
         ametsuchi::TemporaryWsv &temporaryWsv) {
       log_->info("transactions in proposal: {}", proposal.transactions.size());
       auto checking_transaction = [this](auto &tx, auto &queries) {
-        return (queries.getAccount(tx.creator_account_id) |
+
+        return (queries.getAccount(tx.creatorAccountId()) |
                 [&](const auto &account) {
                   // Check if tx creator has account and has quorum to execute
                   // transaction
-                  return tx.signatures.size() >= account.quorum
-                      ? queries.getSignatories(tx.creator_account_id)
+                  return tx.signatures().size() >= account.quorum
+                      ? queries.getSignatories(tx.creatorAccountId())
                       : nonstd::nullopt;
                 }
                 |
                 [&](const auto &signatories) {
+                  auto transaction =
+                      std::unique_ptr<model::Transaction>(tx.makeOldModel());
```

Comment:
Is it possible to use shared_model transaction for fetch signatures?

Reply:
It is already done in #963